modules/node-app-service/main.tf

```diff
@@ -10,7 +10,7 @@ resource "azurerm_linux_web_app" "web_app" {
   service_plan_id               = var.app_service_plan_id
   client_certificate_enabled    = false
   https_only                    = true
-  public_network_access_enabled = false
+  public_network_access_enabled = var.inbound_vnet_connectivity
```

Comment:
Think this will need to be 
public_network_access_enabled = !var.inbound_vnet_connectivity because it's now the inverse of how we creating a private endpoint in the past

Reply:
updated main file with public_network_access_enabled = !var.inbound_vnet_connectivity